src/masternode-payments.cpp

```diff
@@ -33,8 +33,10 @@ CCriticalSection cs_mapMasternodePaymentVotes;
 *   - When non-superblocks are detected, the normal schedule should be maintained
 */
 
-bool IsBlockValueValid(const CBlock& block, int nBlockHeight, CAmount blockReward)
+bool IsBlockValueValid(const CBlock& block, int nBlockHeight, CAmount blockReward, std::string &strErrorRet)
 {
+    strErrorRet = "";
+
     bool isNormalBlockValueMet = (block.vtx[0].GetValueOut() <= blockReward);
     if(fDebug) LogPrintf("block.vtx[0].GetValueOut() %lld <= blockReward %lld\n", block.vtx[0].GetValueOut(), blockReward);
```

Comment:
`fDebug` is `true` if _any_ debug category was set. This line is kind of unrelated to this specific PR which purpose was to get _error message_ done right, can be fixed in some (log) cleanup PR if needed imo since we have more of this, in masternode-sync for example.